vector/src/main/java/im/vector/app/features/roomprofile/settings/RoomSettingsController.kt

```diff
@@ -46,7 +46,7 @@ class RoomSettingsController @Inject constructor(
         fun onNameChanged(name: String)
         fun onTopicChanged(topic: String)
         fun onHistoryVisibilityClicked()
-        fun onAliasChanged(alias: String)
+        fun onOpenAlias()
```

Comment:
I found `openAlias` naming confusing. Is it a term that I don't know?

Reply:
I renamed it to `onRoomAliasesClicked`